### PR TITLE
new file

### PR DESCRIPTION
**Pull request by parikshitgupta1**
new file created as per the steps

### CODE REVIEW

File: _hackers/parikshitgupta.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+name: parikshit
+# What is your GitHub handle?
+github: parikshtgupta1
+# In what role are you submitting, remove the not-matching
+role:
+ - participant
+
+# optionally, you can let us know more about you:
+## What are the pronouns you'd like us to use when talking about you
+pronouns: a07p
```
Comment (gnunicorn) [inline _hackers/parikshitgupta.md]: `pronouns` are "he/him", "she/her", "they/them" or similar...not sure what a07p is ...

Comment (gnunicorn) [reply _hackers/parikshitgupta.md]: one example for you to use:
```suggestion
pronouns: "he/him"
```

Comment (gnunicorn) [reply _hackers/parikshitgupta.md]: or pick this one maybe:

```suggestion
pronouns: "she/her"
```

File: _hackers/parikshitgupta.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Are you on gitter? Put the name here
+gitter:
+## do you have a matrix/riot username?
+matrix:
+## do you have a twitter?
+twitter: Anonymous07P
+
+---
+
+_I can optionally give further information and other ways to access or learn more about me here in full-featured markdown._
```
Comment (gnunicorn) [inline _hackers/parikshitgupta.md]: you probably want to remove this line:
```suggestion

```

File: _hackers/parikshitgupta.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Are you on gitter? Put the name here
+gitter:
+## do you have a matrix/riot username?
+matrix:
+## do you have a twitter?
+twitter: Anonymous07P
+
+---
+
+###https://parikshitgupta.in
```
Comment (riusricardo) [inline _hackers/parikshitgupta.md]:
```suggestion
###https://parikshitgupta.in
```

File: _hackers/parikshitgupta.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+## What are the pronouns you'd like us to use when talking about you
+pronouns: he
+## Are you on gitter? Put the name here
+gitter:
+## do you have a matrix/riot username?
+matrix:
+## do you have a twitter?
+twitter: Anonymous07P
+
+---
```
Comment (riusricardo) [inline _hackers/parikshitgupta.md]:
```suggestion

```

File: _hackers/parikshitgupta.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (riusricardo) [inline _hackers/parikshitgupta.md]:
```suggestion

```

**Pull request merged**